src/NServiceBus.AcceptanceTesting/Support/ScenarioRunner.cs

```diff
@@ -265,6 +265,7 @@ static string GenerateTestTimedOutMessage(TimeSpan maxTime)
             {
                 throw new Exception("Starting endpoints took longer than 2 minutes");
             }
+            await completedTask.ConfigureAwait(false);
```

Comment:
@danielmarbach I already blame you for all the async stuff. My understanding of this code is that we capture the faulted task in `completedTask` variable but since `WhenAny` returns `Task<Task>`, we were not awaiting the `completedTask` because `StartEndpoints` returns a brand new task, not `completedTask`.